### ISSUE TITLE
running on laptop

### ISSUE THREAD

**Issue by AndrewSmith477** [tau-adl/Detection_Tracking_JetsonTX2#15]
May I know if the code can be run on a normal computer?  Would there be any difference when installing the dependent software?

**Comment by Boinet** [comment]
Yes of course. No reason for you not to be able to run it on normal computer

**Comment by RoseDeSable** [comment]
Hi,
 what do you mean with "normal" computer. Under windows I have problems, because I cannot install coffee. But can I run the software under linux on a normal laptop, that has an integrated camera ? Under windows the camera is running as a directshow device. In linux the camera is found under "/dev/..." path. Can the software access it about the dev-path ?

Best Regards
Rose
